backend/src/ml_space_lambda/app_configuration/lambda_functions.py

```diff
@@ -19,10 +19,107 @@
 import time
 
 from ml_space_lambda.data_access_objects.app_configuration import AppConfigurationDAO, AppConfigurationModel, SettingsModel
+from ml_space_lambda.enums import EnvVariable, IAMEffect, IAMStatementProperty, ServiceType
+from ml_space_lambda.utils.account_utils import get_account_arn
 from ml_space_lambda.utils.common_functions import api_wrapper
+from ml_space_lambda.utils.iam_manager import IAM_RESOURCE_PREFIX, IAMManager
+from ml_space_lambda.utils.mlspace_config import get_environment_variables
 
+iam_manager = IAMManager()
 log = logging.getLogger(__name__)
 app_configuration_dao = AppConfigurationDAO()
+env_variables = get_environment_variables()
+
+group_permissions = [
```

Comment:
Why is this called `group_permissions`?

Reply:
These are groups with shared IAM permissions. These permissions cannot be deactivated unless all services in the group are deactivated. Is there a preferred alternate name for this?

Reply:
Updated to `service_group_disable_permissions` so that it more closely resembles the service permissions object naming convention that follows